models/statistic.go

```diff
@@ -37,7 +44,17 @@ func GetStatistic() (stats Statistic) {
 		Count    int64
 		IsClosed bool
 	}
+
 	issueCounts := []IssueCount{}
+	stats.Counter.IssueByLabel = []IssueByLabelCount{}
+
+	_ = db.GetEngine(db.DefaultContext).
+		Select("COUNT(*) AS count, l.name as label").
```

Comment:
Is there maybe a `Count` function inside `Engine`?
If yes, please use that instead.
Ah, I see the potential problem with that:
Then you cannot customize the resulting column names any longer, right?

Reply:
Also, why is the first `AS` uppercase, but the second one lowercase? Please unify, I propose to uppercase.

Reply:
> Is there maybe a `Count` function inside `Engine`? If yes, please use that instead. Ah, I see the potential problem with that: Then you cannot customize the resulting column names any longer, right?

Yes it was to get more explicit with struct `IssueByLabelCount`

> Also, why is the first `AS` uppercase, but the second one lowercase? Please unify, I propose to uppercase.

It's done